pymem/__init__.py

```diff
@@ -78,6 +78,25 @@ def list_modules(self):
         modules = pymem.process.enum_process_module(self.process_handle)
         return modules
 
+    def resolve_offsets(self, base_offset, offsets, *, is_64_bit = True):
```

Comment:
Is `is_64_bit` necessary here? `check_wow64` is run on init of the class - we can use `self.is_WoW64` instead of making this an arg.

Reply:
seems like a good idea to me